test/aws-codeartifact/integ.codeartifact.ts

```diff
@@ -0,0 +1,27 @@
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as ocf from '../../src';
+
+class CodeartifactStack extends cdk.Stack {
+  constructor(scope: Construct) {
+    super(scope, 'codeartifact', { env: { region: 'eu-central-1' } });
```

Comment:
For when other Constructs are added in the future. Also, there should be no need to specify region.

```suggestion
class CodeArtifactStack extends cdk.Stack {
  constructor(scope: Construct) {
    super(scope, 'CodeArtifactDomainAndRepository');
```